src/panels/lovelace/editor/hui-entities-card-row-editor.ts

```diff
@@ -0,0 +1,232 @@
+import { mdiDrag } from "@mdi/js";
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  internalProperty,
+  LitElement,
+  property,
+  PropertyValues,
+  TemplateResult,
+} from "lit-element";
+import { guard } from "lit-html/directives/guard";
+import type { SortableEvent } from "sortablejs";
+import Sortable, {
+  AutoScroll,
+  OnSpill,
+} from "sortablejs/modular/sortable.core.esm";
+import { fireEvent } from "../../../common/dom/fire_event";
+import "../../../components/entity/ha-entity-picker";
+import type { HaEntityPicker } from "../../../components/entity/ha-entity-picker";
+import "../../../components/ha-icon-button";
+import { sortableStyles } from "../../../resources/ha-sortable-style";
+import { HomeAssistant } from "../../../types";
+import { EntityConfig, LovelaceRowConfig } from "../entity-rows/types";
+
+@customElement("hui-entities-card-row-editor")
+export class HuiEntitiesCardRowEditor extends LitElement {
+  @property({ attribute: false }) protected hass?: HomeAssistant;
+
+  @property({ attribute: false }) protected entities?: LovelaceRowConfig[];
+
+  @property() protected label?: string;
+
+  @internalProperty() private _attached = false;
+
+  @internalProperty() private _renderEmptySortable = false;
+
+  private _sortable?: Sortable;
+
+  public connectedCallback() {
+    super.connectedCallback();
+    this._attached = true;
+  }
+
+  public disconnectedCallback() {
+    super.disconnectedCallback();
+    this._attached = false;
+  }
+
+  protected render(): TemplateResult {
+    if (!this.entities || !this.hass) {
+      return html``;
+    }
+
+    return html`
+      <h3>
+        ${this.label ||
+        this.hass!.localize("ui.panel.lovelace.editor.card.generic.entities") +
+          " (" +
+          this.hass!.localize("ui.panel.lovelace.editor.card.config.required") +
+          ")"}
+      </h3>
+      <div class="entities">
+        ${guard([this.entities, this._renderEmptySortable], () =>
+          this._renderEmptySortable
+            ? ""
+            : this.entities!.map((entityConf, index) => {
+                return html`
+                  <div
+                    class="entity"
+                    data-entity-id=${"entity" in entityConf
```

Comment:
What is this used for? If it needs to be unique, type is not the way to go (entity-id also isn't unique as you can have the same entity in a card twice but...)

Reply:
Its for sortable

I think I tried to remove it before and it didn't work. I'll try again